src/viewer.js

```diff
@@ -487,7 +487,9 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      * is the same as the options parameter for {@link OpenSeadragon.Viewer#addTiledImage},
      * except for the index property; images are added in sequence.
      * A TileSource specifier is anything you could pass as the tileSource property
-     * of the options parameter for {@link OpenSeadragon.Viewer#addTiledImage}.
+     * of the options parameter for {@link OpenSeadragon.Viewer#addTiledImage}. To get the TiledImage
```

Comment:
This addition is good, but it doesn't need to be specific to the `tileSources` param. Instead, move it up to the description of this function (line 483).